test/e2e/tests/test_endpoint.py

```diff
@@ -279,10 +290,10 @@ def update_endpoint_failed_test(
         current_config_name = endpoint_resource["status"].get(
             "latestEndpointConfigName"
         )
-        assert (
-            current_config_name is not None
-            and current_config_name
-            == old_config_resource["spec"].get("endpointConfigName", None)
+        assert current_config_name is not None and current_config_name == old_config_resource[
```

Comment:
why these formatting changes, are you using a different tool/setting ?

please use black

Reply:
I am using black not sure why they changed